apko-publish/action.yaml

```diff
@@ -65,6 +60,9 @@ outputs:
   digest:
     description: |
       The digest of the published container image.
+  image-refs:
+    description: |
+      All the image and index refs by digest that were built, newline delimited.
```

Comment:
This is wrong, the logic as-is turns it into a JSON array

Reply:
Can we also phase this in?  Removing the option will immediately break the downstream distroless repos.